.eslintrc.yml

```diff
@@ -1,22 +1,36 @@
 extends:
   - chartjs
   - plugin:markdown/recommended
+  - plugin:@typescript-eslint/recommended
 
-env:
-  es2021: true
-  browser: true
-  node: true
+parser: "@typescript-eslint/parser"
 
 parserOptions:
+  ecmaVersion: 2022
   sourceType: module
-  ecmaFeatures:
-    impliedStrict: true
-    modules: true
 
-plugins: ['html']
+env:
+  es2022: true
+  browser: true
+  node: true
+  jasmine: true
+
+plugins:
+  - "@typescript-eslint"
+  - prettier
+  - html
 
 rules:
+  prettier/prettier: "error"
+  semi: ["error", "never"]
```

Comment:
Is there a specific reason you want to turn semi off?
If so, some files still use them like config.ts for vuepress.

I am more of a fan of using them but that can also be something of getting used to

Reply:
Just a personal preference. Also ok keeping it on.

Reply:
I guess either way is fine.

Approved the pr